tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js

```diff
@@ -52,7 +52,7 @@ test.describe( `$e.run( 'editor/elements/views/widget' )`, () => {
 		expect( emptyViewPlaceholderCount ).toBe( 1 );
 	} );
 
-	test.skip( 'Check if the empty placeholder is displayed inside the Basic Gallery widget', async ( { page }, testInfo ) => {
+	test( 'Check if the empty placeholder is displayed inside the Basic Gallery widget', async ( { page }, testInfo ) => {
```

Comment:
@mykytamurzin I notice these tests still fail because of long loading times after the responsive breakpoints are updated.